Update test for #291

stevenschlansker opened the pull request:
Currently the case is mis-matched which accidentally relies on `final` field setting.
Instead, just fix the annotations so the case matches and the constructor
is properly used

cowtowncoder commented:
I think use of `final` here may be an oversight: would it be possible to instead remove `final` markers to test regular Builder pattern for XML module?
(unless there are already similar tests)

stevenschlansker committed:
Update test for #291

Currently the case is mis-matched which relies on `final` field setting.
Instead, just fix the annotations so the case matches and the constructor
is properly used

stevenschlansker commented:
Ok, does this look better?

cowtowncoder commented:
@stevenschlansker Yes, thank you! I think I'll backport this in 2.19 as well.
